src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import '@src/App.css'
-
+import Background, {Inner} from '@src/components/Background'
+import Title from '@src/components/Title'
+import Banner from '@src/components/Banner'
+import Maintheme from '@src/components/Maintheme'
+import Realtime from './components/Realtime'
 function App() {
```

Comment:
전체적으로 화살표 함수를 사용하셨는데 여기서만 함수 선언식을 사용하셨네요..!
아마 초기 설정 그대로 하신 것 같은데, 함수 유형을 통일하는 것을 권장드릴게요.

```suggestion
const App = () => {
```

ESLint로 지정해줄 수도 있는데요! `react/function-component-definition` 속성 찾아보시고 적용해보셔도 좋을 것 같습니다.